biz.aQute.repository/src/aQute/bnd/deployer/repository/LocalIndexedRepo.java

```diff
@@ -168,7 +236,7 @@ private synchronized void generateIndex(File indexFile, IRepositoryContentProvid
 	}
 
 	@SuppressWarnings("deprecation")
-	private void gatherFiles(Set<File> allFiles) throws Exception {
+	void gatherFiles(Set<File> allFiles) throws Exception {
```

Comment:
It's used in the tests I added, I call it directly to test that the files gathered respect the policy